changelog/1.7.0.md

```diff
@@ -10,3 +10,10 @@ Contrib:
   write `[Structurally]` in `a.isEqual[Structurally](b)`.
 - `(a: Subtype).isEqual(b: Supertype)` will no longer compile, either upcast
   `a: Supertype` or swap the order to `b.isEqual(a)`
+- The `Extract[A, B]` typeclass has been added.
```

Comment:
How about? 

> has been added as a helper to extract `Seq[Stat/Mod]` out of `Defn.{Object,Val,...}`. 

Maybe we should move the tutorial page to the `readme` project in this repo, so we can document new features immediately and link to them in the changelog.